types/site/_data/i18n/authors.d.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare global {
+  export interface I18nAuthor {
+    title: {
+      [lang: string]: string;
+    };
+    description: {
+      [lang: string]: string;
+    };
+  }
+  export interface I18nAuthors {
```

Comment:
I think you have a TODO in the code where you should instead be using this type.

Reply:
I'm not sure, possibly? But id I do it's probably because of some temporary typings issue.